docs/front-end-overview.md

```diff
@@ -185,94 +185,94 @@ The part within the callback would execute once the GET request had returned, wi
 
-The  format for checking for [StyleLint](https://stylelint.io/) violations is:
+The format for checking for [StyleLint](https://stylelint.io/) violations is:
 `./node_modules/stylelint/bin/stylelint.js <file or folder path>`
 
-For example, to find all rule violations in a folder, you can run:
+For example, to find all rule violations in a folder you can run:
 `./node_modules/stylelint/bin/stylelint.js src/encoded/static/scss/`
 
-You can also check just one file (e.g datacolor.js) by  running:
+You can also check just one file (e.g datacolor.js) by running:
 `./node_modules/stylelint/bin/eslint.js src/encoded/static/scss/style.scss`
 
 #### Correcting [StyleLint](https://stylelint.io/) Violations
 
 The format for correcting the [StyleLint](https://stylelint.io/) violations is:
 `./node_modules/stylelint/bin/stylelint.js <file or folder path> --fix`
 
-For example, you fix all rule violations in a folder, by running:
+For example, you fix all rule violations in a folder by running:
 `./node_modules/stylelint/bin/stylelint.js src/encoded/static/scss/ --fix`
 
-You can also fix violation for just one file (e.g style.css) by  running:
+You can also fix violation for just one file (e.g style.css) by running:
 `./node_modules/stylelint/bin/stylelint.js src/encoded/static/components/style.scss -- fix`
 
 Numerous other options can be found at [StyleLint Command line interface Documention page](https://stylelint.io/user-guide/usage/cli)
 
-You can search on [YouTube](https://www.youtube.com/results?search_query=stylelint+) to see if there are any videos that illustrate using the command line with  [StyleLint](https://stylelint.io/).  In general, the idea is similar to  the [ESLint](https://eslint.org/).
+You can search on [YouTube](https://www.youtube.com/results?search_query=stylelint+) to see if there are any videos that illustrate using the command line with [StyleLint](https://stylelint.io/). In general, the idea is similar to the [ESLint](https://eslint.org/).
 #### Editor Extension
 
-Widely used code text editors generally have extensions that allows you to visually detect [StyleLint Command line interface Documention page](https://stylelint.io/user-guide/usage/cli) violations by seeing a squiggly under the code in question. They also display information on what rule was violated by mousing over the squiggly and sometimes automated correction by right clicking and selecting an appropiate option.
+Popular code Editors display information pertaining to [StyleLint Command line interface Documention page](https://stylelint.io/user-guide/usage/cli) violated rule on hover as well as automated corrections, which can be selected and applied by clicking on fix (wording may vary depending on editor).
 
-Here is collection of many extenstions for [popular text editors](https://stylelint.io/user-guide/integrations/editor)
+Here is a collection of extenstions for [popular code editors](https://stylelint.io/user-guide/integrations/editor)
```

Comment:
I think I missed this in the first review - sorry! - "extenstions" should be "extensions". Also in the text above which you did not change "Documention" should be "Documentation".